.env.example

```diff
@@ -1,7 +1,4 @@
-KOVAN_INFURA_URL=
-KOVAN_DEV_PRIVATE_KEY=
-
-MAINNET_INFURA_URL=
-MAINNET_DEV_PRIVATE_KEY=
-
-ETHERSCAN_API_KEY=
\ No newline at end of file
+INFURA_API_KEY=your infura api key
+MNEMONIC=here is where your twelve words mnemonic should be put my friend
```

Comment:
A mnemonic can be longer than 12 words

Reply:
This comes from https://github.com/paulrberg/solidity-template we should push this request upstream if important